### ISSUE TITLE
BullAdapter is not a constructor

### ISSUE THREAD

**Issue by mathewcst** [vcapretz/bull-board#179]
I've just add the package to my project and followed the "Hello World" from the readme and it doesn't work.

I just run the server and get 

```
setQueues([new BullAdapter(registerQueue)])
           ^
TypeError: BullAdapter is not a constructor
```

**Comment by hrydi** [comment]
Hellooo, why you closing this issue?
did you solve it yet?
how you solve it?

thank you

**Comment by felixmosh** [comment]
We didn’t released the latest change (the one that uses adapters) yet.
Should be done today

**Comment by mathewcst** [comment]
Hey, sorry for not providing a fix.

In my case, I literally just removed the `BullAdapter` and passed the queue to the array like: 
```
const myQueue = Queue.add('myQueueName', data)
setQueues([myQueue])
```

**Comment by felixmosh** [comment]
> Hey, sorry for not providing a fix.
> 
> In my case, I literally just removed the `BullAdapter` and passed the queue to the array like:
> 
> ```
> const myQueue = Queue.add('myQueueName', data)
> setQueues([myQueue])
> ```

**Comment by felixmosh** [comment]
We just released v1.1.0 that uses the adapter pattern.

**Comment by hrydi** [comment]
thank you, i will look it later
